web/src/pages/Transaction/index.tsx

```diff
@@ -1,19 +1,467 @@
-import React from 'react'
+import React, { useState, useContext } from 'react'
+import AppContext from '../../contexts/App'
+
 import Page from '../../components/Page'
 import Header from '../../components/Header'
 import Content from '../../components/Content'
 import Footer from '../../components/Footer'
+import { PanelDiv, BriefInfoDiv, InputOutPutTable, WithRowDiv } from './styled'
+
+import BlockHeightIcon from '../../asserts/block_height_green.png'
+import TimestampIcon from '../../asserts/timestamp_green.png'
+import TransactionIcon from '../../asserts/transaction_green.png'
+import VersionIcon from '../../asserts/version.png'
+import CopyGreenIcon from '../../asserts/copy_green.png'
+import CopyIcon from '../../asserts/copy.png'
+
+import { transaction, cell } from './mock'
+
+const cellData = cell.data
+const operationItems = ['Lock Script', 'Type Script', 'Data']
+
+const RowData = ({
+  type,
+  d,
```

Comment:
`d` maybe too simple for a variable name

Reply:
data is better, thanks